includes/results.inc

```diff
@@ -1099,6 +1099,8 @@ class IslandoraSolrFacets {
 
     // Show more link.
     if (count($buckets) > $soft_limit) {
+      // Create hook_islandora_solr_facet_variables_alter().
+      drupal_alter('islandora_solr_facet_variables', $buckets);
```

Comment:
This seems not the best place to apply the alter. Won't trigger if the the buckets are < than the soft limit. To make other changes, would it not be advice-able to work on the theme layer?

Also, it would be great if you could pass to the hook also the facet field. So allow people to have more control/like know who is providing the facets when trying/deciding to alter

Reply:
Moved the alter up a notch.

> Also, it would be great if you could pass to the hook also the facet field.
Would love to. But... I haven't found the variable that contains it..

Reply:
> To make other changes, would it not be advice-able to work on the theme layer?

I originally had my hook in theme.inc, but was advised that I shouldn't put it there.

Reply:
Yeah, theme alter is different, no need for an alter there really, just different processing via your own override, theming works differently. So good there

Reply:
The facet field is:   $facet_field = $this->facet_field;

Reply:
Thanks! I think I got it all.